### PR TITLE
fix(nx-dev): website copy corrections and changes

### PR DESCRIPTION
**Pull request by catfireparty**
Some copy corrections and suggestions for the new homepage.

### CODE REVIEW

File: nx-dev/ui-home/src/lib/work-better-achieve-more-ship-quicker.tsx [hunk 1]
```diff
@@ -16,12 +16,12 @@ export function WorkBetterAchieveMoreShipQuicker(): JSX.Element {
...
           className="scroll-mt-24"
         >
-          Work better together. Achieve more. Ship quicker.
+          Collaborate better. Achieve more. Ship quicker.
         </SectionHeading>
         <SectionHeading as="p" variant="subtitle" className="mt-6">
           <Strong>Tangled codebases</Strong> without well-defined ownership{' '}
           <Strong>crush the velocity</Strong> of teams and the quality of
-          products. Nx solves it. By defining{' '}
+          products. Nx solves the problem. By defining{' '}
```
Comment (juristr) [inline nx-dev/ui-home/src/lib/work-better-achieve-more-ship-quicker.tsx:24]: doesn't "solve it" sound more "snappy"?

**Review by juristr — CHANGES_REQUESTED** [review]

File: nx-dev/ui-home/src/lib/ci-for-monorepos.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export function CiForMonorepos(): JSX.Element {
           </SectionHeading>
           <SectionHeading as="p" variant="subtitle" className="mt-6">
             Current CI systems are <Strong>slow</Strong>,{' '}
-            <Strong>hard to maintain and unreliable</Strong>. With Nx Cloud, we
+            <Strong>hard to maintain, and unreliable</Strong>. With Nx Cloud, we
```
[the remark this answers is not in the document]
Comment (jdpearce) [reply nx-dev/ui-home/src/lib/ci-for-monorepos.tsx:39]: Here it makes sense though, going by the rule of 3s you don't have a 3-group if the comma isn't there?

File: nx-dev/ui-home/src/lib/ci-for-monorepos.tsx [hunk 3]
```diff
@@ -130,8 +130,8 @@ export const ApplicationCard = () => {
           Powerful and elegant UI
         </p>
         <p className="mt-2 text-slate-600 dark:text-slate-400">
-          CI application build for monorepos, so you can quickly find what
-          failed, debug and move on.
+          CI built for monorepos, so you can quickly find what failed, debug and
```
[the remark this answers is not in the document]
Comment (jdpearce) [reply nx-dev/ui-home/src/lib/ci-for-monorepos.tsx:133]: "An application built for monorepo CI, so you can..."?

**Review by juristr — APPROVED** [review]
LGTM, thanks

File: nx-dev/ui-home/src/lib/work-better-achieve-more-ship-quicker.tsx (hunk 1 above)
Comment (jdpearce) [reply nx-dev/ui-home/src/lib/work-better-achieve-more-ship-quicker.tsx]: "Nx is the solution"?

**Pull request merged**